### DISCUSSION TITLE
Response codes for exceeding api usage limits

### DISCUSSION THREAD

**Discussion by mjabl** [deepgram/community#438] [General help] [answered]
Hi, looking at documentation under https://developers.deepgram.com/reference/errors I can't see any errors representing exceeded concurrency / duration limits. What should I expect to happen if I run too many file transcriptions at the same time? What if my transcript takes longer than your limits permit?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @mjabl, thanks for raising this documentation gap. I've updated the [Errors](https://developers.deepgram.com/reference/errors) page that you linked to add that you'll receive a 429 Too Many Requests error if you exceed our rate limits. There are some other possible errors for other reasons that I'll work on adding to that page as well. We allow 20 minutes of processing time for each request before timing out, in which case you can expect to receive a 408 Request Timeout error.
